### ISSUE TITLE
Windows 10 #9926 - Injector not starting

### ISSUE THREAD

**Issue by finndev** [DarthTon/Blackbone#20]
The injector is not starting on the last W10 technical preview build.

.dmp file:

```
Dump Summary
------------
Dump File:	Xenos.exe_2015.03.06.14.52.56.dmp :F:\RamDisk\TMPFS\Xenos-master\Xenos\build\Win32\Debug\Xenos.exe_2015.03.06.14.52.56.dmp
Last Write Time:	3/6/2015 2:53:12 PM
Process Name:	Xenos.exe F:\RamDisk\TMPFS\Xenos-master\Xenos\build\Win32\Debug\Xenos.exe
Process Architecture:	x86
Exception Code:	0xC0000005
Exception Information:	The thread tried to read from or write to a virtual address for which it does not have the appropriate access.
Heap Information:	Present

System Information
------------------
OS Version:	10.0.9926
CLR Version(s):	

Modules
-------
Module Name	Module Path	Module Version
-----------	-----------	--------------
Xenos.exe	F:\RamDisk\TMPFS\Xenos-master\Xenos\build\Win32\Debug\Xenos.exe	2.1.2.0
ntdll.dll	C:\Windows\System32\ntdll.dll	10.0.9926.0
kernel32.dll	C:\Windows\System32\kernel32.dll	10.0.9926.0
KERNELBASE.dll	C:\Windows\System32\KERNELBASE.dll	10.0.9926.0
apphelp.dll	C:\Windows\System32\apphelp.dll	10.0.9926.0
bcrypt.dll	C:\Windows\System32\bcrypt.dll	10.0.9926.0
shlwapi.dll	C:\Windows\System32\shlwapi.dll	10.0.9926.0
msvcrt.dll	C:\Windows\System32\msvcrt.dll	7.0.9926.0
combase.dll	C:\Windows\System32\combase.dll	10.0.9926.0
rpcrt4.dll	C:\Windows\System32\rpcrt4.dll	10.0.9926.0
sspicli.dll	C:\Windows\System32\sspicli.dll	10.0.9926.0
CRYPTBASE.dll	C:\Windows\System32\CRYPTBASE.dll	10.0.9926.0
bcryptPrimitives.dll	C:\Windows\System32\bcryptPrimitives.dll	10.0.9926.0
sechost.dll	C:\Windows\System32\sechost.dll	10.0.9926.0
gdi32.dll	C:\Windows\System32\gdi32.dll	10.0.9926.0
user32.dll	C:\Windows\System32\user32.dll	10.0.9926.0
comdlg32.dll	C:\Windows\System32\comdlg32.dll	10.0.9926.0
SHCore.dll	C:\Windows\System32\SHCore.dll	10.0.9926.0
dbghelp.dll	C:\Windows\System32\dbghelp.dll	10.0.9926.0
shell32.dll	C:\Windows\System32\shell32.dll	10.0.9926.0
advapi32.dll	C:\Windows\System32\advapi32.dll	10.0.9926.0
ole32.dll	C:\Windows\System32\ole32.dll	10.0.9926.0
MSVCP120D.dll	C:\Windows\System32\MSVCP120D.dll	12.0.21005.1
MSVCR120D.dll	C:\Windows\System32\MSVCR120D.dll	12.0.21005.1
comctl32.dll	C:\Windows\WinSxS\x86_microsoft.windows.common-controls_6595b64144ccf1df_6.0.9926.0_none_2203b4f072dcfff0\comctl32.dll	6.10.9926.0
imm32.dll	C:\Windows\System32\imm32.dll	10.0.9926.0
msctf.dll	C:\Windows\System32\msctf.dll	10.0.9926.0
oleaut32.dll	C:\Windows\System32\oleaut32.dll	10.0.9926.0
```

Is anyone experiencing the same issue?

**Comment by DarthTon** [comment]
Can you please upload dump + pdb + binary somewhere?

**Comment by DarthTon** [comment]
Ah, yes. Apischema structures were changed and now it crashes after eating all available memory.

**Comment by DarthTon** [comment]
Fixed in 9a3441157b1ded2c89d42c5c714409099e8ae206
